operate/client/e2e-playwright/tests/resources/ProcessInstanceMigration/orderProcessMigration_v_3.bpmn

```diff
@@ -446,14 +446,38 @@
     <bpmn:sequenceFlow id="Flow_06bvbdi" sourceRef="SubProcess" targetRef="Gateway_5" />
     <bpmn:sequenceFlow id="Flow_0up5li4" sourceRef="MultiInstanceSubProcess2" targetRef="Gateway_5" />
     <bpmn:sequenceFlow id="Flow_0xc7gwl" sourceRef="EscalationSubProcess" targetRef="Gateway_5" />
-    <bpmn:sequenceFlow id="Flow_19stf58" sourceRef="EscalationEvent" targetRef="Gateway_5" />
+    <bpmn:sequenceFlow id="Flow_19stf58" sourceRef="EscalationBoundaryEvent" targetRef="Gateway_5" />
     <bpmn:sequenceFlow id="Flow_1tyhemw" sourceRef="Gateway_6" targetRef="EndEvent_1" />
     <bpmn:parallelGateway id="Gateway_6">
       <bpmn:incoming>Flow_1ibxab9</bpmn:incoming>
       <bpmn:incoming>Flow_1c49dph</bpmn:incoming>
       <bpmn:outgoing>Flow_1tyhemw</bpmn:outgoing>
     </bpmn:parallelGateway>
     <bpmn:sequenceFlow id="Flow_1c49dph" sourceRef="Gateway_5" targetRef="Gateway_6" />
+    <bpmn:subProcess id="EscalationEventSubProcess" name="Escalation event sub process" triggeredByEvent="true">
+      <bpmn:endEvent id="Event_1cmlzk4">
+        <bpmn:incoming>Flow_0of03mm</bpmn:incoming>
+      </bpmn:endEvent>
+      <bpmn:serviceTask id="EscalationEventTask" name="Escalation event task">
+        <bpmn:extensionElements>
+          <zeebe:taskDefinition type="escalationEventTask" />
+        </bpmn:extensionElements>
+        <bpmn:incoming>Flow_0vd8mzp</bpmn:incoming>
+        <bpmn:outgoing>Flow_0of03mm</bpmn:outgoing>
+      </bpmn:serviceTask>
+      <bpmn:startEvent id="EscalationStartEvent" name="Escalation start event" isInterrupting="false">
+        <bpmn:outgoing>Flow_0vd8mzp</bpmn:outgoing>
+        <bpmn:escalationEventDefinition id="EscalationEventDefinition_0wxb74s" escalationRef="Escalation_1fppfbl" />
+      </bpmn:startEvent>
+      <bpmn:sequenceFlow id="Flow_0of03mm" sourceRef="EscalationEventTask" targetRef="Event_1cmlzk4" />
+      <bpmn:sequenceFlow id="Flow_0vd8mzp" sourceRef="EscalationStartEvent" targetRef="EscalationEventTask" />
+    </bpmn:subProcess>
+    <bpmn:intermediateThrowEvent id="EscalationThrowEvent">
+      <bpmn:incoming>Flow_0ykwgtn</bpmn:incoming>
+      <bpmn:outgoing>Flow_0bhp20l</bpmn:outgoing>
+      <bpmn:escalationEventDefinition id="EscalationEventDefinition_1aqu025" escalationRef="Escalation_1fppfbl" />
+    </bpmn:intermediateThrowEvent>
+    <bpmn:sequenceFlow id="Flow_0bhp20l" sourceRef="EscalationThrowEvent" targetRef="EscalationSubProcess" />
```

Comment:
Only change in the diagrams is an added escalation sub process containing an escalation start event and a service task.